image/image.go

```diff
@@ -464,12 +477,18 @@ func setBootvars(downloadedSnapsInfo map[string]*snap.Info) error {
 		"snap_try_core":   "",
 		"snap_try_kernel": "",
 	}
+
 	for _, fn := range snaps {
 		bootvar := ""
 
-		info := downloadedSnapsInfo[fn]
+		info := downloadedSnapsInfoForBootConfig[fn]
 		switch info.Type {
-		case snap.TypeOS:
+		// FIXME: workaround a store bug thatsends
+		// "type: base" as "application" :/
+		// Once the store is fixed the next two lines can be removed
```

Comment:
This will be easier to remove if you include the URL of the bug.

Reply:
Thanks, fixed by removing the code (store got fixed since)